deploy/helm/magda/charts/gateway/values.yaml

```diff
@@ -26,3 +26,51 @@ resources: {}
   # requests:
   #  cpu: 100m
   #  memory: 128Mi
+routes: 
+  search:
+    to: http://search-api/v0
+  registry:
+    to: http://registry-api/v0
+    auth: true
+  auth:
+    to: http://authorization-api/v0/public
+    methods:
+    - GET
+    auth: true
+  discussions:
+    to: http://discussions-api/v0
+    auth: true
+  admin:
+    to: http://admin-api/v0
+    auth: true
+  feedback:
+    to: http://feedback-api/v0
+  correspondence:
+    to: http://correspondence-api/v0/public
+helmet: {}
+csp:
+  browserSniff: false
+  directives:
+    scriptSrc:
+    - "''self''"
+    - "''unsafe-inline''" # For VWO, hopefully we can remove this soon
+    - "''data:''" # Also for VWO
```

Comment:
A very minor thing:
The `data:` should be included in CSP without single quote `'`
![image](https://user-images.githubusercontent.com/674387/42675023-750439ae-86b5-11e8-8188-57dfd23178c1.png)